resources/plugins/arty2/plugin.json

```diff
@@ -0,0 +1,38 @@
+{
+    "nameInUI": "Carsten's Arty Spotter for Single Player",
+    "defaultValue": false,
+    "specificOptions": [
+        {
+            "nameInUI": "Time to impcat of the rounds",
+            "mnemonic": "user_fireDelay",
+            "defaultValue": 15
+        },
+        {
+            "nameInUI": "Salvo quantity",
+            "mnemonic": "user_quantity",
+            "defaultValue": 5
+        },
+        {
+            "nameInUI": "Impact radius",
+            "mnemonic": "user_spread",
+            "defaultValue": 150
+        },
+        {
+            "nameInUI": "Max spotting distance. In kilometers.", 
+            "mnemonic": "user_spottingDistance",
+            "defaultValue": 15
+        }
+    ],
+    "scriptsWorkOrders": [
+        {
+            "file": "CG_ArtySpotter_V1_2a_MP.lua",
+            "mnemonic": "arty"
+        }
+    ],
+    "configurationWorkOrders": [
+        {
+            "file": "arty-config.lua",
+            "mnemonic": "arty-config"
```

Comment:
I'm not entirely sure we can re-use the same name for the arty-config mnemonic, do you mind double checking if this doesn't cause any issues? 😅